include/circt/Dialect/SV/SVStatements.td

```diff
@@ -474,3 +481,27 @@ def CoverOp : SVOp<"cover", [ProceduralOp]> {
   let results = (outs);
   let assemblyFormat = "attr-dict $property `:` type($property)";
 }
+
+def BindOp : SVOp<"bind", [IsolatedFromAbove, HasParent<"mlir::ModuleOp">]> {
```

Comment:
Oh, maybe this just has other stuff mixed in?  BindOp should (is?) definitely a separable thing.

Reply:
Yeah, this patch set is stacked onto https://github.com/llvm/circt/pull/982. The `BindOp` stuff should land first. GitHub annoyingly has no good support for indicating this. :(

Reply:
Aha, got it.  Let's get it in! :-)